data-azure-cosmos/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id "io.micronaut.build.internal.data-module"
+    id 'io.micronaut.test-resources'
+}
+
+micronaut {
+    version = '3.7.3'
+    testResources {
+        enabled = true
+        inferClasspath = false
+    }
+}
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
```

Comment:
I don't know what is the best approach for new modules, without disabling this way, the build was failing trying to compare with default version 3.3.0 which is not available yet.

Reply:
That’s correct

Reply:
Thanks for confirming.